VAMobile/src/components/BackButton.tsx

```diff
@@ -59,8 +55,6 @@ export const BackButton: FC<BackButtonProps> = ({
     return null
   }
 
-  const chevron = showCarat ? <VAIcon mt={1} name={'ChevronLeft'} fill="backButton" testID="BackButtonCarat" /> : <></>
```

Comment:
with us using our own screen templates, the back button in headerStyles is technically not used, but in the off chance we do forget to have a template applied to a screen I opted to keep the component and just remove the icon.

Reply:
see prior comment in back button